Frontend/ui-driver/src/Screens/HomeScreen/Handler.purs

```diff
@@ -195,6 +195,8 @@ homeScreen = do
     SwitchPlan plan updatedState -> do 
       modifyScreenState $ HomeScreenStateType (\_ -> updatedState)
       App.BackT $ App.BackPoint <$> (pure $ SWITCH_PLAN_FROM_HS plan updatedState)
+    GotoHotspotScreen updatedState -> do
+      App.BackT $ App.NoBack <$> (pure $ GOTO_HOTSPOT_SCREEN updatedState)
```

Comment:
Bro can you update the global state here? else it will create bugs in future.

Reply:
not required bro